docs/src/community/phrasebook.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: ../common_links.inc
+
+.. _phrasebook:
+
+Package Phrasebook
+===================
+
+There are a number of similar packages to Iris, and a lot of these have their own
+terminology for similar things. Whether you're coming or going, we hope this might
+be a helpful guide to these differences!
+Definitions for each can be found in :ref:`glossary`. See also
+`Xarray terminology <https://docs.xarray.dev/en/stable/user-guide/terminology.html>`_.
+
+.. list-table:: Phrasebook
+   :widths: 25 25 25 50
+   :header-rows: 1
+
+   * - Iris
+     - Xarray
+     - Example
+     - Notes
+   * - Non-Lazy
+     - Eager
+     -
+     - Used to relate to functions, rather than the data.
+   * - Cube
+     - DataArray
+     -
+     -
+   * - Cubelist
+     - Dataset
+     -
+     -
+   * - Merge
+     - Concatenate
+     -
+     - Functionality is essentially swapped.
+   * - Concatenate
+     - Merge
+     -
+     - Functionality is essentially swapped.
```

Comment:
From what I've read of the xarray documentation, I'm not sure it's quite as simple as marge and concatenate being swapped versions of what they are in iris. The way I read it,  xarray concatenate can act like iris merge but it can also act like iris concatenate. It looks like xarray marge is not quite equivalent to either merge or concatenate since its output is a Dataset, and while Iris has CubeList, this lacks some of the features of a Dataset so I don't think it would be possible to have something that functions quite like xarray merge in Iris.